src/index.js

```diff
@@ -15,10 +15,10 @@ const rangeQueryParameters = Object.freeze({
 });
 
 class WakaTimeClient {
-  constructor(apiKey) {
+  constructor(apiKey, apiUrl) {
     this.apiKey = apiKey;
     this.axiosConfiguration = axios.create({
-      baseURL: 'https://wakatime.com/api/v1/',
+      baseURL: apiUrl || 'https://wakatime.com/api/v1/',
```

Comment:
Do you have a preference for this approach vs. using [default parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters)?

One benefit of using default parameters is that falsy values other than `undefined` would result in some type of error to the client user vs. using the default url.

In terms of browser-compatibility, while default parameters are not supported by IE, [the current babel configuration](https://github.com/jaebradley/wakatime-client/blob/f50c0f8e5f9cc9b7a988c539810fd4eae0282076/.babelrc#L18) should transpile everything down to ES5.

Reply:
Frankly, I didn't know about default parameters, until now I thought they were only a thing in TypeScript. Thanks for pointing that out! I'll change it.